tests/ci/clickbench.py

```diff
@@ -53,7 +53,7 @@ def get_run_command(
     env_str = " ".join(envs)
 
     return (
-        f"docker run --volume={builds_path}:/package_folder "
+        f"docker run --shm-size=16g --volume={builds_path}:/package_folder "
```

Comment:
It should be something like `max(16, ram-x)` to not overgrow, and being able to run anywhere

Reply:
The data size is around 13 GB.